orchagent/dash/dashvnetorch.cpp

```diff
@@ -37,13 +37,15 @@ extern size_t gMaxBulkSize;
 extern CrmOrch *gCrmOrch;
 extern Directory<Orch*> gDirectory;
 
-DashVnetOrch::DashVnetOrch(DBConnector *db, vector<string> &tables, ZmqServer *zmqServer) :
+DashVnetOrch::DashVnetOrch(DBConnector *db, vector<string> &tables, DBConnector *app_state_db, ZmqServer *zmqServer) :
```

Comment:
Would you clarify where does this specify about APP_STATE_DB on NPU?

Reply:
Added a comment in main.cpp where these DBs are instantiated and also provided related PRs including DBConnector support for these tables in the PR description.